assemblies/resources/system.properties.append

```diff
@@ -3,6 +3,7 @@
 net.sf.ehcache.skipUpdateCheck=true
 org.terracotta.quartz.skipUpdateCheck=true
 
-# Configuring DocumentBuilderFactory and TransformerFactory
-javax.xml.parsers.DocumentBuilderFactory=org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
-javax.xml.transform.TransformerFactory=org.apache.xalan.processor.TransformerFactoryImpl
+# Configure for native JDK Impls
+javax.xml.parsers.DocumentBuilderFactory=com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderFactoryImpl
+javax.xml.parsers.SAXParserFactory=com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl
+javax.xml.transform.TransformerFactory=com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl
```

Comment:
@karendolan Just found that replacing the xalan stuff here causes https://opencast.jira.com/browse/MH-13311. I verified that undo the change of line 9 fixes MH-13311. Not clear to me whether this requires the re-introduction of some dependencies you removed in the POMs... does it?

Reply:
Hi @staubesv, it's possible that changing back the TransformerFactory without changing the capture-admin and message-broker dependencies back could affect those 2 modules. However,  the coverimage module did not have difficulty finding TransformerFactory and did not have a declared dependency before. Our site is running in production with this change, but we haven't run the coverimage workflow. 

In short, it would be safest to revert all but line 8. Meanwhile, I am curious and will investigate the "serious configuration errors" that the coverimage module has with the Java8 TransformerFactory.